runtimes/standalone/src/lib.rs

```diff
@@ -906,6 +895,93 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl did_rpc_runtime_api::DidApi<
```

Comment:
Since it's used in all runtimes, why don't we move it into common? So far we have the same features deployed in all runtimes.

Reply:
yeah it's a huge chunk of code. I would like to slim it down a bit but wasn't sure how.
The issue is, since it's implemented for the runtime you can't move it out there.
The impl also has to be in the `impl_runtime_apis!` block.

Reply:
I could move the following to commons since it's shared between all queries.

```rust
let accounts = pallet_did_lookup::ConnectedAccounts::<Runtime>::iter_key_prefix(&owner_info.owner).collect();
let service_endpoints = did::ServiceEndpoints::<Runtime>::iter_prefix(&owner_info.owner).map(|e|From::from(e.1)).collect();
did_rpc_runtime_api::RawDidDocument {
  identifier: owner_info.owner,
  w3n: Some(name.into()),
  accounts,
  service_endpoints,
  details: details.into(),
}
```